db/schema.rb

```diff
@@ -10,7 +10,35 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_12_10_084736) do
+ActiveRecord::Schema.define(version: 2020_12_16_013702) do
+
+  create_table "active_storage_attachments", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
```

Comment:
I think your active storage stuff snuck in here

Reply:
Fixed